src/main/kotlin/cacophony/codegen/BlockLabel.kt

```diff
@@ -1,3 +1,15 @@
 package cacophony.codegen
 
+import cacophony.semantic.syntaxtree.Definition
+import kotlin.math.absoluteValue
+
 data class BlockLabel(val name: String)
+
+fun functionBodyLabel(function: Definition.FunctionDeclaration): BlockLabel {
+    return BlockLabel(
+        when (function.identifier) {
+            "<program>" -> "start"
+            else -> "${function.identifier}_${function.arguments.size}_${function.hashCode().absoluteValue}"
```

Comment:
I do not like the idea of appending hashCode() to asm labels. I think they should be easy to predict based on the function declaration.

Reply:
Now suffix of a label is a unique id